src/kouta_indeksoija_service/indexer/indexer.clj

```diff
@@ -45,6 +45,10 @@
   [entries]
   (set (remove nil? (mapcat tutkinnonosa-ids-on-koulutus entries))))
 
+(defn quick-index-koulutukset [oids execution-id]
```

Comment:
Kannattaako näitä monikkomuotoisia funktioita tehdä jos niitä ei kuitenkaan käytetä apista?